apps/site/assets/css/_schedule-page-line-diagram.scss

```diff
@@ -67,118 +106,234 @@ $line-width: 10px;
     }
   }
 
+  // "View schedules" link
   .m-schedule-diagram__footer {
     text-align: center;
 
     .btn {
-      padding: unset;
+      padding: 0;
     }
   }
 
+  // renders blue (link color) alert icons instead of black
   .c-svg__icon-alerts-triangle path:not(:nth-child(1)) {
     fill: currentColor;
   }
 
-  //
-  //  Schedule line diagram
-  //
   //  .m-schedule-diagram__lines contains all the
   //  graphics for the line diagram, and uses negative
   //  margins to counter the padding from its parent
   //  .m-schedule-diagram__stop.
   //  A .m-schedule-diagram__line is drawn for each
-  //  branch, and creates the line. Stop circles are SVGs.
+  //  branch, and creates the line.
   .m-schedule-diagram__lines {
     display: flex;
     margin: calc(-#{$base-spacing} - #{$border-width}*2) 0;
     margin-right: $base-spacing;
```

Comment:
Looks like we also need this right margin to shrink to .5rem also to match the previous change: 

**CURRENT**
![image](https://user-images.githubusercontent.com/5806643/72447469-60a6a980-3783-11ea-84ae-e08e3c2240a9.png)
**DESIRED**
![image](https://user-images.githubusercontent.com/5806643/72447492-6ac8a800-3783-11ea-9e28-612646885655.png)